src/berry_blender.c

```diff
@@ -313,6 +281,526 @@ static void sub_805197C(u16 a0, u16 a1);
+static const u8 sUnknownJpnString5[] = _("カシコイ"); // "smart"
+static const u8 sUnknownJpnString6[] = _("タクマシ"); // "tough" (missing an イ at the end)
+
+static const u8 gUnknown_08216249[] = _("\p");
+
+// unreferenced; These appear to be the first names of four people who worked on the game.
+static const u8 sUnknownJpnString7[10] = _("てつじ"); // Tetsuji (Ohta)
+static const u8 sUnknownJpnString8[10] = _("あきと"); // Akito (Mori)
+static const u8 sUnknownJpnString9[10] = _("シゲル"); // Shigeru (Ohmori)
+static const u8 sUnknownJpnString10[10] = _("ヨシノリ"); // Yoshinori (Matsuda)
+
+static const u8 sUnknownText_2Pok[] = _("2Pok");
+static const u8 sUnknownText_3Pok[] = _("3Pok");
+static const u8 sUnknownText_4Pok[] = _("4Pok");
+static const u8* const gUnknown_08216284[] =
+{
+      sUnknownText_2Pok, sUnknownText_3Pok, sUnknownText_4Pok
+};
+
+// unreferenced player strings
+static const u8 sUnknown1PString[4] = _("1P");
+static const u8 sUnknown2PString[4] = _("2P");
+static const u8 sUnknown3PString[4] = _("3P");
+static const u8 sUnknown4PString[4] = _("4P");
+
+#ifdef ENGLISH
+static const u8 sBlenderOpponentName1[] = _("MISTER");
+static const u8 sBlenderOpponentName2[] = _("LADDIE");
+static const u8 sBlenderOpponentName3[] = _("LASSIE");
+#else // GERMAN
+static const u8 sBlenderOpponentName1[] = _("OPI");
+static const u8 sBlenderOpponentName2[] = _("KUMPEL");
+static const u8 sBlenderOpponentName3[] = _("TUSSI");
+#endif // ENGLISH
+static const u8* const sBlenderOpponentsNames[] =
+{
+    sBlenderOpponentName1, sBlenderOpponentName2, sBlenderOpponentName3
+};
+
+static const u8 sRedColorString[] = _("{COLOR RED}");
+static const u8 sNewLineString_0[] = _("\n");
+static const u8 sSpaceString_0[] = _(" ");
+
+static const s8 gUnknown_082162CC[][2] =
+{
+    {-1, -1}, {1, -1}, {-1, 1}, {1, 1}
+};
+
+static const u8 gUnknown_082162D4[][2] =
+{
+    {2, 6}, {23, 6}, {2, 12}, {23, 12}, {1, 6}, {22, 6}, {1, 12}, {22, 12}
+};
+
+static const u8 sBlenderSyncArrowsPos[][2] =
+{
+    {72, 32}, {168, 32}, {72, 128}, {168, 128}
+};
+
+static const u8 gUnknown_082162EC[3][4] =
+{
+    {-1, 0, 1, -1}, {-1, 0, 1, 2}, {0, 1, 2, 3}
+};
+
+static const u16 gUnknown_082162F8[] = {0, 0xC000, 0x4000, 0x8000};
+static const u8 gUnknown_08216300[] = {1, 1, 0};
+static const u8 gUnknown_08216303[] = {32, 224, 96, 160, 0};
+
+static const TaskFunc gUnknown_08216308[] =
+{
+    &sub_804F8C8, &sub_804F9F4, &sub_804FB1C
+};
+
+static const struct OamData sOamData_8216314 =
+{
+    .y = 0,
+    .affineMode = 0,
+    .objMode = 0,
+    .mosaic = 0,
+    .bpp = 0,
+    .shape = 0,
+    .x = 0,
+    .matrixNum = 0,
+    .size = 2,
+    .tileNum = 0,
+    .priority = 1,
+    .paletteNum = 0,
+    .affineParam = 0,
+};
+
+static const union AnimCmd sSpriteAnim_821631C[] =
+{
+    ANIMCMD_FRAME(16, 5, 1, 1),
+    ANIMCMD_END
+};
+
+static const union AnimCmd sSpriteAnim_8216324[] =
+{
+    ANIMCMD_FRAME(16, 5, 0, 1),
```

Comment:
You can do `ANIMCMD_FRAME(16, 5, .vFlip = TRUE);` to make it more readable. That's how I intended the macro to be used.